<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	/*background-color: rgb(207, 242, 255);*/
	background-color: rgb(155, 207, 212);
	border: None;
	font-size: 9pt;
}

QPushButton {
	border-radius: 10px;
	padding-top: 4px;
	padding-bottom: 4px;
}

QComboBox:focus{
	border: 2px solid black;
}

QComboBox{
	height: 20px;
	border-radius: 14px;
	background-color: rgb(100, 207, 212);
	border: 2px solid rgb(100, 207, 212);
	padding: 2px 10px;
}


QComboBox::drop-down{
	border: None;
}

QComboBox::down-arrow{
	width: 20px;
	margin-right: 20px;
}

#frame_2 {
	border: 2px solid black;
	padding: 7px;
	border-radius: 15px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QCustomSlideMenu" name="LeftMenuContainer" native="true">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: 2px solid black;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>15</number>
          </property>
          <property name="leftMargin">
           <number>11</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="generate_btn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Générer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add_btn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Ajouter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delete_btn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Supprimer</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>560</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="mainBodyContainer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="mainBodyContent" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCustomStackedWidget" name="mainPages">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page_generation">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QFrame" name="frame_2">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <layout class="QGridLayout" name="gridLayout">
                   <property name="horizontalSpacing">
                    <number>100</number>
                   </property>
                   <property name="verticalSpacing">
                    <number>20</number>
                   </property>
                   <item row="0" column="1">
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>Mot Clé :</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Auteur :</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QComboBox" name="auteurs_list"/>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="label_3">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Difficulté :</string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="0">
                    <widget class="QComboBox" name="difficultes_list">
                     <item>
                      <property name="text">
                       <string/>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>1</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>2</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>3</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QComboBox" name="keywors_list"/>
                   </item>
                   <item row="3" column="1">
                    <widget class="QCheckBox" name="corrige_checkbox">
                     <property name="text">
                      <string>Corrigé</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_4">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <property name="spacing">
                  <number>40</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QTableWidget" name="table">
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 211, 212);</string>
                   </property>
                   <column>
                    <property name="text">
                     <string>Réf exo</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Fichier</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Auteur</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Difficulté</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Mot Clés</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Corrigé</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Auteur Corrigé</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_3">
                <property name="styleSheet">
                 <string notr="true">QPushButton {
	border: 2px solid black;
	height: 22px;
}</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <property name="spacing">
                  <number>90</number>
                 </property>
                 <property name="leftMargin">
                  <number>50</number>
                 </property>
                 <property name="topMargin">
                  <number>10</number>
                 </property>
                 <property name="rightMargin">
                  <number>50</number>
                 </property>
                 <property name="bottomMargin">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="ajouter">
                   <property name="text">
                    <string>Ajouter des exercices</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="clean_btn">
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="text">
                    <string>Nettoyer la table</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="creation_btn">
                   <property name="font">
                    <font>
                     <family>MS Shell Dlg 2</family>
                     <pointsize>9</pointsize>
                     <weight>50</weight>
                     <bold>false</bold>
                    </font>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="text">
                    <string>Créer le fichier</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="add_page">
             <widget class="QListWidget" name="listWidget">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>40</y>
                <width>391</width>
                <height>561</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 170, 0);</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="delete_page"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomSlideMenu</class>
   <extends>QWidget</extends>
   <header location="global">Custom_Widgets.Widgets.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">Custom_Widgets.Widgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
